lib/workers/branch/index.ts

```diff
@@ -363,134 +354,14 @@ export async function processBranch(
       getAdminConfig().trustLevel === 'high' &&
```

Comment:
@rarkins Should we also remove this here?
```suggestion
```

Reply:
Done, I moved all the "Should postupgradecommands run" logic into `lib/workers/branch/execute-post-upgrade-commands.ts`